src/v2/Apps/Show/routes.tsx

```diff
@@ -85,6 +85,12 @@ export const routes: RouteConfig[] = [
           }
         `,
       },
+      {
+        path: "hours",
+        render: props => {
+          throw new RedirectException(`/show/${props.match.params.slug}`, 302)
+        },
+      },
```

Comment:
From other examples, I thought I'd be able to use `new Redirect` here. That doesn't seem to work nor do the other spots where that approach is used.

For example:

```typescript
new Redirect({
  from: "*",
  to: "/artist/:artistID",
}) as any
```

https://github.com/artsy/force/blob/master/src/v2/Apps/Artist/routes.tsx#L327-L335